fs/littlefs/Kconfig

```diff
@@ -6,17 +6,69 @@ config FS_LITTLEFS
 		Build the LITTLEFS file system. https://github.com/littlefs-project/littlefs.
 
 if FS_LITTLEFS
-config FS_LITTLEFS_BLOCK_FACTOR
-	int "LITTLEFS Block size multiple factor"
+config FS_LITTLEFS_PROGRAM_SIZE_FACTOR
+	int "LITTLEFS Program size multiplication factor"
+	default 1
+	---help---
+		A factor used for multiplying program size.
+
+		The result is used as the minimum size of a block program in bytes.
+		All program operations will be a multiple of the result.
+
+config FS_LITTLEFS_READ_SIZE_FACTOR
+	int "LITTLEFS Read size multiplication factor"
+	default FS_LITTLEFS_PROGRAM_SIZE_FACTOR
+	---help---
+		A factor used for multiplying read size.
+
+		The result is used as the minimum size of a block read in bytes.
+		All read operations will be a multiple of the result.
+
+config FS_LITTLEFS_BLOCK_SIZE_FACTOR
+	int "LITTLEFS Block size multiplication factor"
 	default 4
 	---help---
-		Configure the cache size of the LITTLEFS file system with a multiple factor of the block size.
```

Comment:
change the default to 1. If we keep the old defsult setting, FS_LITTLEFS_PROGRAM_SIZE_FACTOR defsult value should to 4 instead

Reply:
@Igor-Misic this comment isn't addressed yet.

Reply:
@xiaoxiang781216 I hope I've tackled all your suggestions. If not feel free to use "add suggestion" from GitHub directly with values that I didn't set properly.